dimod/binary/binary_quadratic_model.py

```diff
@@ -649,42 +662,39 @@ def add_linear_inequality_constraint(
     ) -> Iterable[Tuple[Variable, int]]:
         """Add a linear inequality constraint as a quadratic objective.
 
-        Adds a linear inequality constraint of the form:
-
-        math:'lb <= \sum_{i,k} a_{i,k} x_{i,k} + constant <= ub'
+        Adds a linear inequality constraint of the form,
```

Comment:
Should this be consistent with the first line? I.e. `Add` instead of `Adds`? Applies other places as well.

Reply:
We usually don't for the "more elaborate description" that follows the one-sentence summary line; e.g., 
https://docs.ocean.dwavesys.com/en/stable/docs_system/reference/generated/dwave.system.samplers.DWaveSampler.sample_ising.html. PEP 257 requires that the summary line prescribes as a command but doesn't say the same for multi-line docstings.
If the second use of the same verb jars, can rephrase